airflow/providers/amazon/aws/hooks/dms.py

```diff
@@ -0,0 +1,222 @@
+        create_task_kwargs: Optional[dict] = None,
+    ) -> str:
+        """
+        Create DMS replication task
+
+        :param replication_task_id: Replication task id
+        :type replication_task_id: str
+        :param source_endpoint_arn: Source endpoint ARN
+        :type source_endpoint_arn: str
+        :param target_endpoint_arn: Target endpoint ARN
+        :type target_endpoint_arn: str
+        :param replication_instance_arn: Replication instance ARN
+        :type replication_instance_arn: str
+        :param table_mappings: Table mappings
+        :type table_mappings: dict
+        :param migration_type: Migration type ('full-load'|'cdc'|'full-load-and-cdc'), full-load by default.
+        :type migration_type: str
+        :param create_task_kwargs: Extra arguments for DMS replication task creation.
+        :type create_task_kwargs: Optional[dict]
+        :return: Replication task ARN
+        """
+        additional_args = create_task_kwargs or {}
+        dms_client = self.get_conn()
+        create_task_response = dms_client.create_replication_task(
+            ReplicationTaskIdentifier=replication_task_id,
+            SourceEndpointArn=source_endpoint_arn,
+            TargetEndpointArn=target_endpoint_arn,
+            ReplicationInstanceArn=replication_instance_arn,
+            MigrationType=migration_type,
+            TableMappings=json.dumps(table_mappings),
+            **additional_args,
+        )
+
+        replication_task_arn = create_task_response['ReplicationTask']['ReplicationTaskArn']
+        self.wait_for_task_status(replication_task_arn, 'ready')
+
+        return replication_task_arn
+
+    def start_replication_task(
+        self,
+        replication_task_arn: str,
+        start_replication_task_type: str,
+        start_task_kwargs: Optional[dict] = None,
+    ):
+        """
+        Starts replication task.
+
+        :param replication_task_arn: Replication task ARN
+        :type replication_task_arn: str
+        :param start_replication_task_type: Replication task start type
+            ('start-replication'|'resume-processing'|'reload-target')
+        :type start_replication_task_type: str
+        :param start_task_kwargs: Extra start replication task arguments
+        :type start_task_kwargs: Optional[dict]
+        """
+        additional_args = start_task_kwargs or {}
+        dms_client = self.get_conn()
+        dms_client.start_replication_task(
+            ReplicationTaskArn=replication_task_arn,
+            StartReplicationTaskType=start_replication_task_type,
+            **additional_args,
+        )
+
+    def stop_replication_task(self, replication_task_arn):
+        """
+        Stops replication task.
+
+        :param replication_task_arn: Replication task ARN
+        :type replication_task_arn: str
+        """
+        dms_client = self.get_conn()
+        dms_client.stop_replication_task(ReplicationTaskArn=replication_task_arn)
+
```

Comment:
Similar to not using `running` waiter in start task, I have opted out to not use stopped waiter here, since it seems to not work in a straightforward way.

Running and stopped waiters in DMS have an unusual behaviour that is unintuitive for users:
https://github.com/boto/botocore/blob/develop/botocore/data/dms/2016-01-01/waiters-2.json#L168
https://github.com/boto/boto3/issues/1926